hw4.Rmd

```diff
@@ -30,43 +35,75 @@ Before doing anything else, just play with the data to get a sense of it.
 (@) The main outcome of interest is *decision* which records whether the judge granted an appeal or not.  In what proprotion of cases did the judge grant an appeal.
 
 ```{r}
-# [your code here]
+yes <- GS_data %>%
+  filter(decision == "yes") %>%
+  summarise(yes = n())
+  
+prop_yes <- yes/length(GS_data$decision)
+prop_yes
 ```
 
 (@) There are 12 different judges in the data. A key question is whether different judges have different rates of granting an appeal.  Make a plot showing the number of cases per judge.
 
 ```{r}
-# [your code here]
+cases <- GS_data %>%
+  group_by(judge) %>%
+  summarise(cases = n())
+cases
+
+p <- ggplot(data=cases, aes(x=judge, y=cases)) + geom_bar(stat="identity") + labs(x="Judge", y="Cases") + ggtitle("Cases per Judge")
```

Comment:
Its a little confusing that both your data frame and y variable have the same name